### PR TITLE
Dwc support

### PR DESCRIPTION
**Pull request by d047491**

### CODE REVIEW

File: pkg/uploader/archiver.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ type archiveContent struct {
 	file   string
 }
 
-func (up *uploader) createArchive(log *libbuildpack.Logger, root string) (io.Reader, error) {
+func CreateArchive(log *libbuildpack.Logger, root string) (io.Reader, error) {
```
Comment (jkbschmid) [inline pkg/uploader/archiver.go:24]: if you prefer, we can revert the changes in this file

**Review by jkbschmid — COMMENTED** [review]
Only some documentation changes, otherwise LGTM

File: README.md [hunk 2]
```diff
@@ -8,11 +8,25 @@ This is a supply/sidecar buildpack which can't be used stand-alone. It has two m
 Consume the latest released version of this buildpack with the following link in your manifest or via the `-b` flag:
 
 https://github.com/SAP/cloud-authorization-buildpack/releases/latest/download/opa_buildpack.zip
-We discourage referencing a branch of this repo directly.
+Referencing a branch of this repository directly, increases the staging time of the app.
```
[the remark this answers is not in the document]
Comment (d047491) [reply README.md:11]: maybe we should exacty these to reasons, for transparency

[the remark this answers is not in the document]
Comment (jkbschmid) [reply README.md:11]: yes. Let's address this in a separate PR that also adds a warning for everyone that's using "main"

[the remark this answers is not in the document]
Comment (d047491) [reply README.md]: I just committed a change reverting and adding reason, why we discourage

**Review by jkbschmid — APPROVED** [review]
